src/elements/Input/Input.js

```diff
@@ -97,6 +98,10 @@ const Input = ({
 	}
 
 	if(type === 'textarea') {
```

Comment:
This needs to happen not just for the `textarea` but the default input as well (the last "default" return in this control flow)